src/frontend/hooks/hasAnticheatInfo.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react'
+import { AntiCheatInfo, GameInfo } from 'common/types'
+
+export const hasAnticheatInfo = (gameInfo: GameInfo) => {
```

Comment:
custom hook so we can use it in different places without repeating the fetch logic